### PR TITLE
Server certificate verification for Mesos components.

### PR DESCRIPTION
**Pull request by lava**
## High-level description

This PR bumps Mesos to a version that permits server certificate verification to be enabled in the corresponding EE PR.

## Corresponding DC/OS tickets (required)

  - [DCOS_OSS-5345](https://jira.mesosphere.com/browse/DCOS_OSS-5345) Bump Mesos to the version with all the "proper TLS" goodies.

## Checklist for component/package updates:

If you are changing components or packages in DC/OS (e.g. you are bumping the sha or ref of anything underneath `packages`), then in addition to the above please also include:

  - [ ] Change log from the last version integrated (this should be a link to commits for easy verification and review): [example](https://github.com/dcos/dcos-mesos-modules/compare/f6fa27d7c40f4207ba3bb2274e2cfe79b62a395a...6660b90fbbf69a15ef46d0184e36755881d6a5ae)
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain.
         Included in DC/OS Enterprise, since the change will only affect enterprise strict mode by default.
  - [ ] Test Results: [link to CI job test results for component]
  - [ ] Code Coverage (if available): [link to code coverage report]

### CODE REVIEW

**Comment by lava** [comment]
@mesosphere-mergebot bump-ee

**Comment by lava** [comment]
@mesosphere-mergebot bump-ee

**Comment by lava** [comment]
@mesosphere-mergebot bump-ee

**Comment by lava** [comment]
@mesosphere-mergebot label "Ready for Review"

File: CHANGES.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 
 * Added L4LB metrics in DC/OS Net. (DCOS_OSS-5011)
 
+* Bumped Mesos to upstream commit '1a6760c60dc823b088ffbcf48909cf3e371570f3'. (DCOS_OSS-5342)
```
Comment (jgehrcke) [inline CHANGES.md]: No needed :) will update this in an editorial round, no worries.

Can discuss offline if we want

**Comment by tillt** [comment]
@mesosphere-mergebot sync

**Comment by lava** [comment]
@mesosphere-mergebot sync

**Pull request merged**